gradle-diff-task-dependencies/plugin/src/main/groovy/com/novoda/gradle/diffdependencytask/DiffDependencyTaskConfig.groovy

```diff
@@ -0,0 +1,11 @@
+package com.novoda.gradle.diffdependencytask
+
+import com.novoda.gradle.diffdependencytask.git.ChangedFilesProvider
+
+class DiffDependencyTaskConfig {
+
+    public GroovyCallable<List<String>> changedFilesProvider = ChangedFilesProvider.create()
+
+    public ConditionalDependencyRepository conditionalDependencyRepository = new MemoryConditionalDependencyRepository()
```

Comment:
I still fail to see how this helps testability: this object is only holding 2 fields that could be inside the plugin itself 😕
(I second the choice of Liskov-ing them tho)